site/src/docs/components/header/code.mdx

```diff
@@ -304,7 +304,95 @@ import { Header, IconSearch } from 'hds-react';
 
 </Playground>
 
-**Note:** Header is designed to be used full width at the top of the page above the main content. Preview here might not scale correctly, <Link openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="/storybook/react/?path=/story/components-header--with-full-features-dark-theme">view full header with dark theme example in Storybook</Link>.
+#### Custom theme
+
+<Playground scope={{ Header, IconSearch }}>
+
+```jsx
+import { Header, IconSearch } from 'hds-react';
+
+() => (
+  <Header theme={{
+    '--color-focus-outline': 'var(--color-metro-dark)',
```

Comment:
The focus outline only works on SkipLink here as well. Probably will be fixed here automatically when you fix it in scss files.

Reply:
fixed